src/components/Card.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import Proptypes from "prop-types";
+import "./card.css";
+function Card(props) {
+    const { children, className, ...rest } = props;
```

Comment:
The extraction you did in line 5, we can directly extract all props in line 4 itself.
For eg function Button({ type = "default", children, icon, ...rest })